text/0092-logs-vision.md

```diff
@@ -73,6 +73,15 @@ Unified collection is important for the following reasons:
   of resources where the data comes from or by user-defined attributes),
   enabling correct correlation across Resource dimensions later on the backend.
 
+#### Performance
+
+The unified collector must be performant; its resource utilization should be minimal for most use cases, but must also handle high throughput when needed.
```

Comment:
I think this is not necessary since it is already covered in Collector vision doc: https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/vision.md#performant

>Highly stable and performant under varying loads. Well-behaved under extreme load, with predictable, low resource consumption.